spec/jsSpecs/ares-device.spec.js

```diff
@@ -215,11 +227,11 @@ describe(aresCmd + ' negative TC', function() {
     });
 
     it('Capture with invalid destiation Path', function(done) {
-        exec(cmd + ` -c /rootDir`, function (error, stdout, stderr) {
+        exec(cmd + ` -c ${noPermDirPath}`, function (error, stdout, stderr) {
             if (stderr && stderr.length > 0) {
                 common.detectNodeMessage(stderr);
-                expect(stderr).toContain("ares-device ERR! [syscall failure]: EACCES: permission denied, mkdir '/rootDir'");
-                expect(stderr).toContain("ares-device ERR! [Tips]: No permission to execute. Please check the directory permission </rootDir>");
+                expect(stderr).toContain("ares-device ERR! [syscall failure]: EACCES: permission denied, open");
```

Comment:
I think it would be better without 'open'

Reply:
Okay!